### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: cluster/clusterpb/cluster.pb.go [hunk 1]
```diff
@@ -579,6 +579,6 @@ func skipCluster(dAtA []byte) (n int, err error) {
 }
 
 var (
-	ErrInvalidLengthCluster = fmt.Errorf("proto: negative length found during unmarshaling")
+	ErrInvalidLengthCluster = fmt.Errorf("proto: negative length found during unmarshalling")
```
Comment (jsoref) [inline cluster/clusterpb/cluster.pb.go]: This repository is split ~60:40 in favor of doubled `l` over single `l`. There's one API which uses single `l`, and thus that can't be changed.

I can of course drop any commit, so if you want to be inconsistent, that's fine....

File: test/with_api_v1/acceptance/send_test.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ receivers:
 
 	co.Want(Between(6, 6.5), Alert("alertname", "test").Active(5.3))
 
-	// Test against a bug which ocurrec after a restart. The previous occurrence of
+	// Test against a bug which occurred after a restart. The previous occurrence of
```
Comment (jsoref) [inline test/with_api_v1/acceptance/send_test.go]: I'm not certain this is the right part of speech...

File: cluster/clusterpb/cluster.pb.go (hunk 1 above)
Comment (jsoref) [reply cluster/clusterpb/cluster.pb.go]: Dropped because this is actually talking to a third party module.

File: config/config_test.go [hunk 3]
```diff
@@ -374,7 +374,7 @@ func TestMarshalSecretURL(t *testing.T) {
 	// u003c -> "<"
 	// u003e -> ">"
 	require.Equal(t, "\"\\u003csecret\\u003e\"", string(c), "SecretURL not properly elided in JSON.")
-	// Check that the marshaled data can be unmarshaled again.
+	// Check that the marshaled data can be unmarshalled again.
```
Comment (simonpasquier) [inline config/config_test.go]: Both are valid, sticking to one is preferred and the Go documentation uses `marshaled`.

Comment (jsoref) [reply config/config_test.go]: I've updated a branch `spelling-marshal-x`: https://github.com/jsoref/alertmanager/compare/spelling..jsoref:spelling-marshal-x -- If that's what you want, I'll force push.

**Comment by simonpasquier** [comment]
Thanks!

**Pull request merged**